### ISSUE TITLE
Use Docker: Run to start a local registry server

### ISSUE THREAD

**Issue by shizn** [Microsoft/vscode-docker#119]
Sometimes developers might use below command to start a local registry for testing purpose. 
`docker run -d -p 5000:5000 --restart=always --name registry registry:2`

Actually a registry is an instance of the registry image, and runs within Docker. So we are thinking to enable this in **Docker: Run** command.

**Comment by chrisdias** [comment]
What are you proposing to enable? I want to keep the commands as simple as possible with as few questions as possible.  For example, the reason I don't prompt for ports with `docker run` is because it is easier to express the mapping in a `docker-compose.yml` file. So, instead of doing a `docker run`, you would do a `docker-compose up`.

**Comment by shizn** [comment]
Let me rephrase that. I am proposing that `docker run` can support to run a image that has not been downloaded to local machine. In current implementation, Docker:Run can only start local images. However, in docker CLI, if CLI is unable to find the image locally, it can automatically pull down the image from docker hub. The official `registry` image is a good example. Any thoughts?
